small update to main readme (#20)
temporalio/background-checks: README.md

# Background Check application

This is an example [Temporal Application](https://docs.temporal.io/docs/content/what-is-a-temporal-application) project.

The application is written using the [Temporal Go SDK](https://github.com/temporalio/sdk-go).

> The use case for this application is inspired by the blog post 
> ["How Temporal simplified Checkr workflows"](https://docs.temporal.io/blog/how-temporal-simplified-checkr-workflows/)

Reviewer: ```suggestion
This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
```

## How to use the application

- [Guide to using this application](./docs/how-to-use.md)

> ## Learn about application cores, requirements, and design/implementation decisions

Reviewer: ```suggestion
## Learn about the application
```

# Background Checks Application

> This project showcases the implementation of a background checks application with Temporal.
> The application is written in the [Go programming language](https://go.dev/), and uses the 
> Temporal [Go SDK](https://github.com/temporalio/sdk-go).
> See the list of available Temporal SDKs [here](https://docs.temporal.io/). 

Reviewer: ```suggestion
This is an example [Temporal Application](https://docs.temporal.io/docs/content/what-is-a-temporal-application) project.

The application is written using the [Temporal Go SDK](https://github.com/temporalio/sdk-go).
```

The use case for this application is inspired by the blog post 

> ["How Temporal simplified Checkr workflows"](https://docs.temporal.io/blog/how-temporal-simplified-checkr-workflows/)

Reviewer: I actually would prefer not to change this.

The project narrative goes into the inspiration - we want to call out what the use case is directly to align expectations up front.

Reviewer: I meant, we should keep this:
```
This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
```
